src/cms/templates/pages/page_pdf.html

```diff
@@ -17,16 +17,24 @@
         for detailed information about the applied library concepts see
         the documentation of xhtml2pdf -->
         <link href="{% static 'pdf.css' %}" rel="stylesheet" type="text/css">
-        {% if language.slug not in prevent_italics %}
-            <!-- TODO In the moment the only font-family we could find that supports:
-            xhtml2pdf, arabic letters and italic style is DejaVu
-            but DejaVu has the drawback that it does not support for:
-            arabic and italic at the same time, so italic is not available for arabic -->
-            <style type="text/css">
-                em {font-family: DejaVu-Oblique;}
-                pdftoc.pdftoclevel2 {font-family: DejaVu-Oblique;}
-            </style>
-        {% endif %}
+        <style>
+            {% if language.slug == "zh-CN" %}
+                html, body {
+                    font-family: "Noto Sans SC", sans-serif;
+                }
+            {% elif right_to_left %}
+                html, body {
+                    font-family: "DejaVu", sans-serif;
+                }
+            {% else %}
```

Comment:
I don't know, I didn't include any font just because it's used in WordPress. If it is, then it's a coincidence.
I wanted to use the fonts we're agreed on as [Integreat standard](https://wiki.tuerantuer.org/integreat-languages#fonts_for_alphabets) (which doesn't include DejaVu at all), so the only reason for this font to be used in the PDF rendering is that the default font for right-to-left alphabets in the apps (Lateef) isn't supported by our PDF library xhtml2pdf and thus we need a fallback. I mentioned this multiple times in the code, e.g. [here](https://github.com/Integreat/integreat-cms/pull/759/files/8f968aee743d2eb0efd0ef57fa6426b623a34066#diff-4b84d677c44cf2c49b7bb29d20482db4c5b49b19c4881d656df358418b1270a9R5) and [here](https://github.com/Integreat/integreat-cms/pull/759/files/8f968aee743d2eb0efd0ef57fa6426b623a34066#diff-f02d17d496d5e77113ea485502bc9edb26071364e28ed5f44b858bef4a43a7e7R40).